src/net_processing.cpp

```diff
@@ -779,6 +779,23 @@ void PeerLogicValidation::InitializeNode(CNode *pnode) {
         PushNodeVersion(pnode, connman, GetTime());
 }
 
+// We don't force an attempt to relay unbroadcast txns by directly adding them
+// to setInventoryTxToSend, but instead pass to PushInventory which checks
+// filterInventoryKnown.
+//
+// Passing txns through the bloom filter has a stronger privacy model, but weaker
+// delivery guarantees. Under certain network & node conditions, it may still
+// take an extended period of time for the transaction to be initially broadcast
+// to the network.
+void PeerLogicValidation::QueueUnbroadcastTxs(CNode* pnode)
+{
+    LOCK(mempool.cs);
+    for(const uint256& txid : mempool.m_unbroadcast_txids){
```

Comment:
in commit 2f3f449dc9c89420a444cc8263c5a5fe537a475b:

Please use m_mempool and not the global.

Reply:
method no longer exists, but fixed in `ReattemptInitialBroadcast`